Dockerfile-builder.template

```diff
@@ -265,6 +265,8 @@ RUN set -eux; \
 		CONFIG_FEATURE_UTMP \
 		CONFIG_FEATURE_WTMP \
 {{ ) else "" end -}}
+		CONFIG_SHA1_HWACCEL \
+		CONFIG_SHA256_HWACCEL \
```

Comment:
This is fair, but I'd like to also include a comment about this being temporary (so it's more clear when it's safe to remove):

```suggestion
# disable SHA hardware acceleration (temporarily), as it fails with SIGILL on some of GitHub's common CI systems; https://bugs.busybox.net/show_bug.cgi?id=15236
		CONFIG_SHA256_HWACCEL CONFIG_SHA1_HWACCEL \
```

I'm happy to take over from here and make this change (and run `apply-templates.sh`) for you if you'd perfer. :bow: